web/payum.js

```diff
@@ -1,227 +0,0 @@
-/**
- * @callback requestCallback
```

Comment:
shouldn't be removed but moved to public folder

Reply:
Done.